### DISCUSSION TITLE
spaCy integration for Label Studio

### DISCUSSION THREAD

**Discussion by tim-smart** [explosion/spaCy#11220] [Show & Tell]
Github: https://github.com/tim-smart/label-studio-spacy

This project integrates your spaCy pipelines into Label Studio.

This allows you to:

* Get predictions from spaCy to speed up annotation of your data
* Quickly train models without leaving Label Studio
* Tighten the annotation -> training loop to quick iterate on your model

YouTube demo: https://youtu.be/F19NT-21uT4

**Comment by polm** [comment]
Thanks for the submission, looks like a cool project!
